simpeg/potential_fields/gravity/survey.py

```diff
@@ -13,10 +13,30 @@ class Survey(BaseSurvey):
     """
 
     def __init__(self, source_field, **kwargs):
-        self.source_field = validate_type(
-            "source_field", source_field, SourceField, cast=False
+        if "source_list" in kwargs:
+            raise TypeError("source_list is not a valid argument to gravity.Survey.")
```

Comment:
It would be nice to guide users how to set sources in this survey.
```suggestion
            raise TypeError("source_list is not a valid argument to gravity.Survey. Use source_field instead.")
```